### PR TITLE
Make some log lines report at DEBUG level

### LINKED ISSUE #1167: Please improve severity of some log entries
**Is your feature request related to a problem? Please describe.**

When setting the log to level "info" (which seems a reasonable level for production systems: You want some generic info, but not all the debug stuff), there are still a lot of log entries, that shouldn't be in a production system. Most notably entries containing plain text conversations. Those should not end up in the log under normal conditions.

**Describe the solution you'd like**

Please improve the severity of the log levels

**Describe alternatives you've considered**
Not really.

**Additional context**
Here's a suggestion
I have only done the relevant changes to the `.js` files. So this might not work as expected. Consider this as hints!

```diff
--- a/src/bridge/IrcBridge.ts
+++ b/src/bridge/IrcBridge.ts
@@ -670,7 +670,7 @@ export class IrcBridge {
             const data = req.getData();
             const dir = data && data.isFromIrc ? "I->M" : "M->I";
             const duration = " (" + req.getDuration() + "ms)";
-            log.info(`[${req.getId()}] [${dir}] ${msg} ${duration}`);
+            log.debug(`[${req.getId()}] [${dir}] ${msg} ${duration}`);
         }
 
         // SUCCESS
@@ -946,7 +946,7 @@ export class IrcBridge {
             log.error(line);
         }
         else {
-            log.info(line);
+            log.debug(line);
         }
     }
 
--- a/src/bridge/IrcHandler.ts
+++ b/src/bridge/IrcHandler.ts
@@ -564,7 +564,7 @@ export class IrcHandler {
 
         const failed = [];
         for (const room of matrixRooms) {
-            req.log.info(
+            req.log.debug(
                 "Relaying in room %s", room.getId()
             );
             try {
--- a/src/bridge/MatrixHandler.ts
+++ b/src/bridge/MatrixHandler.ts
@@ -514,7 +514,7 @@ export class MatrixHandler {
         // for each room (which may be on different servers)
         ircRooms.forEach((room) => {
             if (room.server.claimsUserId(user.getId())) {
-                req.log.info("%s is a virtual user (claimed by %s)",
+                req.log.debug("%s is a virtual user (claimed by %s)",
                     user.getId(), room.server.domain);
                 return;
             }
@@ -771,7 +771,7 @@ export class MatrixHandler {
         * Matrix --> Matrix (Admin room)
         */
 
-        req.log.info("%s usr=%s rm=%s body=%s",
+        req.log.debug("%s usr=%s rm=%s body=%s",
             event.type, event.sender, event.room_id,
             (event.content.body ? event.content.body.substring(0, 20) : "")
         );
@@ -788,7 +788,7 @@ export class MatrixHandler {
         const servers = this.ircBridge.getServers();
         for (let i = 0; i < servers.length; i++) {
             if (servers[i].claimsUserId(event.sender)) {
-                req.log.info("%s is a virtual user (claimed by %s)",
+                req.log.debug("%s is a virtual user (claimed by %s)",
                     event.sender, servers[i].domain);
                 return BridgeRequestErr.ERR_VIRTUAL_USER;
             }
@@ -849,7 +849,7 @@ export class MatrixHandler {
 
         ircRooms.forEach((ircRoom) => {
             if (ircRoom.server.claimsUserId(event.sender)) {
-                req.log.info("%s is a virtual user (claimed by %s)",
+                req.log.debug("%s is a virtual user (claimed by %s)",
                     event.sender, ircRoom.server.domain);
                 return;
             }
```

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #1167

### CODE REVIEW

**Comment by Half-Shot** [comment]
Reminder to myself that "Server doesn't mirror parts" should not be logged for each room.

**Comment by Half-Shot** [comment]
Also log eventIds, not messages.

File: src/bridge/IrcHandler.ts [hunk 1]
```diff
@@ -217,14 +217,14 @@ export class IrcHandler {
             req.log.error("Cannot route PM to %s - no client", toUser);
             return BridgeRequestErr.ERR_DROPPED;
         }
-        req.log.info("onPrivateMessage: %s from=%s to=%s action=%s",
-            server.domain, fromUser, toUser,
-            JSON.stringify(action).substring(0, 80)
+        req.log.info("onPrivateMessage: %s from=%s to=%s",
```
Comment (jaller94) [inline src/bridge/IrcHandler.ts:220]: What's the reasoning to keep these on info?

**Review by benparsons — APPROVED** [review]
lgtm

File: src/bridge/IrcHandler.ts (hunk 1 above)
Comment (Half-Shot) [reply src/bridge/IrcHandler.ts:220]: It's still really useful to keep track of when people perform actions, the info level just strips away the message data.

**Pull request merged**